Yank/yank.py

```diff
@@ -188,7 +188,9 @@ def ions_atoms(self):
         # Ions are all atoms of the solvent whose residue name show a charge.
         return [i for i in self._solvent_atoms
                 if '-' in self._topology.atom(i).residue.name or
-                '+' in self._topology.atom(i).residue.name]
+                '+' in self._topology.atom(i).residue.name or
+		'NA' in self._topology.atom(i).residue.name or
+		'CL' in self._topology.atom(i).residue.name]
```

Comment:
Thanks for the PR! A couple of suggestions:

1. It looks like you've configured your IDE to use tabs instead of spaces. We shouldn't mix the two types of indentation in Python. Can you switch to 4-space tabs the settings?
2. Using the `in` operator here risks to collect things we don't want. What about having maintaining a set of residue names we want to consider ions (beside the ones containing + and -) and have something like
```python
ION_RESIDUE_NAMES = {'NA', 'CL'}
return [i for i in self._solvent_atoms
    if '-' in self._topology.atom(i).residue.name or
    '+' in self._topology.atom(i).residue.name or
    self._topology.atom(i).residue.name in ION_RESIDUE_NAMES]
```
This way we check for equality instead of looking in the substrings.